brushfire-scalding/src/main/scala/com/stripe/brushfire/scalding/Example.scala

```diff
@@ -17,17 +20,45 @@ class IrisJob(args: Args) extends TrainerJob(args) {
         val values = parts.tail.map { s => s.toDouble }
         Instance(line, 0L, Map(cols.zip(values): _*), Map(label -> 1L))
       }
+  }
 
-  implicit val stopper = FrequencyStopper[String](10, 3)
-  val error = BrierScoreError[String, Long]
-  val trainer =
-    Trainer(trainingData, KFoldSampler(4))
+  def train(trainingData: TypedPipe[Instance[String, Double, Map[String, Long]]]): Trainer[String, Double, Map[String, Long]] = {
+    val trainer = Trainer(trainingData, KFoldSampler(4))
       .expandTimes(args("output"), 3)
       .expandInMemory(args("output") + "/mem", 10)
+
+    trainer
       .validate(error) { results =>
-        results.map { _.toString }.writeExecution(TypedTsv(args("output") + "/bs"))
+        results.map {
+          _.toString
+        }.writeExecution(TypedTsv(args("output") + "/bs"))
       }
+
+    trainer
       .featureImportance(error) { results =>
-        results.map { _.toString }.writeExecution(TypedTsv(args("output") + "/fi"))
+        results.map {
+          _.toString
+        }.writeExecution(TypedTsv(args("output") + "/fi"))
       }
+    trainer
+  }
+
+  val trainingData = load_data()
+
+  def trainer() = {
+    args.optional("load") match {
```

Comment:
guessing you're just using this for debugging and you'll revert it back to always doing the full sequence of things before merging? (For simplicity)